### ISSUE TITLE
Pytorch Histogram Code

### ISSUE THREAD

**Issue by fahadshamshad** [codeslake/Color_Transfer_Histogram_Analogy#3]
Hi, thanks for your great work. Can you upload the pytorch histogram scripts used for training the network (for histogram loss function). Thanks.

**Comment by codeslake** [comment]
I don't exactly remember where I got the code, but here it is.

<script src="https://gist.github.com/codeslake/bfacd42ad37ea4e9c4edaa695f4ee737.js"></script>

**Comment by fahadshamshad** [comment]
Thanks a lot. I am closing the issue. Did you train with batch size 1?

**Comment by codeslake** [comment]
Yes we used 1 for the batch size.
I don't exactly remember, but it took about 2 weeks.

Best,
Closing the issue.
